.env.example

```
@@ -8,6 +8,10 @@ PORT=5000
 
 DEBUG=false
 
+# logging into files inside LNBITS_DATA_FOLDER
+LOG_FILE=true
```

```suggestion
ENABLE_LOG_TO_FILE=true
```
- `LOG_FILE` var name suggests that it represents the name of the log file